experimental/LieAlgebras/test/LieAlgebraModule-test.jl

```diff
@@ -1122,4 +1122,315 @@
+      @test demazure_character(weyl_vector(R), W([1,3])) == dict
+
+      #x=s[3]*s[2]*s[1]
+      dict = Dict([1, 1, 1] => 1,
+                  [-1, 2, 1] => 1,
+                  [-1, 3, -1] => 1,
+                  [1, 2, -1] => 1)
+      @test demazure_character(fundamental_weight(R, 1), W([3,2,1])) == dict
+      dict = Dict([0, 0, 0] => 1,
+                  [0, 1, -2] => 1,
+                  [-1, 1, 0] => 1,
+                  [1, 0, 0] => 1,
+                  [0, -1, 2] => 1)
+      @test demazure_character(fundamental_weight(R, 2), W([3,2,1])) == dict
+      dict = Dict([0, 1, 0] => 1,
+                  [1, -1, 2] => 1,
+                  [1, 0, 0] => 1,
+                  [1, 1, -2] => 1)
+      @test demazure_character(fundamental_weight(R, 3), W([3,2,1])) == dict
+      dict = Dict([2, 0, 1] => 1,
+                  [1, 2, -3] => 1,
+                  [1, 1, -1] => 1,
+                  [1, 1, 1] => 1,
+                  [0, 3, -3] => 1,
+                  [0, 1, 1] => 1,
+                  [-1, 3, -1] => 1,
+                  [2, -1, 3] => 1,
+                  [0, 0, 3] => 1,
+                  [1, -2, 5] => 1,
+                  [0, 2, -1] => 1,
+                  [-1, 2, 1] => 1,
+                  [1, -1, 3] => 1,
+                  [2, 2, -3] => 1,
+                  [1, 2, -1] => 1,
+                  [1, 0, 1] => 1,
+                  [1, 3, -5] => 1,
+                  [2, 1, -1] => 1)
+      @test demazure_character(weyl_vector(R), W([3,2,1])) == dict
+
+      #x=s[3]*s[2]*s[3]*s[1]*s[2]*s[3]*s[1]*s[2]
+      dict = Dict([1, 0, 0] => 1,
+                  [-1, 1, 0] => 1,
+                  [0, -1, 2] => 1,
+                  [0, 0, 0] => 1,
+                  [1, -1, 0] => 1,
+                  [0, 1, -2] => 1)
+      @test demazure_character(fundamental_weight(R, 1), W([3,2,3,1,2,3,1,2])) == dict
+      dict = Dict([-1, 2, -2] => 1,
+                  [1, 1, -2] => 1,
+                  [-2, 1, 0] => 1,
+                  [2, -1, 0] => 1,
+                  [0, -1, 0] => 1,
+                  [-1, -1, 2] => 1,
+                  [-1, 1, 0] => 1,
+                  [1, -2, 2] => 1,
+                  [1, 0, 0] => 1,
+                  [-1, 0, 2] => 1,
+                  [1, -1, 0] => 1,
+                  [0, 1, 0] => 1,
+                  [0, 1, -2] => 1,
+                  [1, -1, 2] => 1,
+                  [-1, 1, -2] => 1,
+                  [0, -1, 2] => 1,
+                  [0, 0, 0] => 3,
+                  [1, 0, -2] => 1,
+                  [-1, 0, 0] => 1)
+      @test demazure_character(fundamental_weight(R, 2), W([3,2,3,1,2,3,1,2])) == dict
+      dict = Dict([0, -1, 1] => 1,
+                  [0, 0, 1] => 1,
+                  [-1, 0, 1] => 1,
+                  [0, 0, -1] =>1,
+                  [-1, 1, -1] => 1,
+                  [0, 1, 1] => 1,
+                  [1, -1, 1] => 1)
+      @test demazure_character(fundamental_weight(R, 3), W([3,2,3,1,2,3,1,2])) == dict
```

Comment:
magma's result for this contains three weight spaces. So I assume this is just a copy-past error while converting to a test here